Face/FindSimilar/Program.cs

```diff
@@ -0,0 +1,198 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Azure.CognitiveServices.Vision.Face;
+using Microsoft.Azure.CognitiveServices.Vision.Face.Models;
+
+namespace FindSimilar
+{
+    public class Program
+    {
+        // Create a client.
+        public static string ApiKey = "ENTER YOUR KEY HERE";
+
+        public static IFaceClient Client = new FaceClient(new ApiKeyServiceClientCredentials(ApiKey))
+        {
+            Endpoint = "ENTER YOUR ENDPOINT HERE"
+        };
+
+        public static void Main(string[] args)
+        {
+            FindSimilar_in_FaceIds();
+            FindSimilar_in_FaceList();
+            FindSimilar_in_LargeFaceList();
+            Console.ReadLine();
+        }
+
+        public static void FindSimilar_in_FaceIds()
+        {
+            List<string> imageFileNames =
```

Comment:
I suggest to divide source and target images as findsimilar in `FaceList/LargeFaceList`.

Reply:
done